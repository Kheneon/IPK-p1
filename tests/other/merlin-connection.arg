-h 147.229.176.19 -p 10002