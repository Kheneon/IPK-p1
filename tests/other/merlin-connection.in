HELLO
SOLVE (* 3 3)
BYE
