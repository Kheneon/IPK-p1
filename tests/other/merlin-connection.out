HELLO
RESULT 9
BYE
